src/common/components/Result/Result.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import styles from './Result.module.scss'
+// import Container from 'common/components/Container/index.js'
+
+export const Result = ({upc, name, rating, img, price, unit, compare, newprod, sameday, pricedrop}) => {
+  const currency = (val) => {
+    let currency = ((val/100).toFixed(2)).toString()
+    currency = currency.split('.')
+    return(
+      <>
+        <span className={styles[`result__price--dollars`]}>
+          ${currency[0]}
+        </span>
+        <span className={styles[`result__price--sr`]}>
+          .
+        </span>
+        <span className={styles[`result__price--cents`]}>
+        {currency[1]}
+        </span>
+      </>
+    )
+  }
+
+  const imgStyle = {
+    backgroundImage: `url(${img})`,
+  }
+
+  return ( 
+  <li 
+    className={styles.result}
+    data-upc={upc}
+    data-pricedrop={pricedrop}
+    data-new={newprod}
+    data-sameday={sameday}
+    data-compare={compare}
```

Comment:
Are you going to use these data attributes in future somewhere?

Reply:
Yeah -- I'm planning to use them if I add the filter functionality